DL_NegotiationRequests.cpp

```diff
@@ -0,0 +1,143 @@
+#include <iostream>
+#include <string>
+
+
+enum class MessageIntentUse {
+      WhenSpeed ,
+      WhenSpeedToSpeed
+    
+};
+
+enum class MessageElement {
+    URG,
```

Comment:
it's better to use the full words 
`URGENT
ALERT
RESPONSE
`

Reply:
I will